templates/layout.html

```diff
@@ -210,7 +118,7 @@
   </footer>
 </template>
 
-<template name="_loginButtonsAdditionalLoggedInDropdownActions">
+<template name="_loginButtonsAdditionalLoggedInDropdownActions"> //TODO This is unused.
```

Comment:
Can you have `//` style comments in a template?? This template is used by ianmartorell/meteor-accounts-ui-bootstrap-3, see https://github.com/ianmartorell/meteor-accounts-ui-bootstrap-3/#add-additional-logged-in-actions.

Maybe we should add a comment explaining who uses this =)